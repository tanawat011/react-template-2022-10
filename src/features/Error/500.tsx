export const Error500 = () => {
  return <div>500 Internal Server Error</div>
}
