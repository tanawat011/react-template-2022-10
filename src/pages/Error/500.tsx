import { PATH } from 'Routes'
import { Button } from 'components/Button'

export const Error500 = () => {
  return (
    <div>
      <p>500 Internal Server Error</p>

      <Button link to={`${PATH.BACKOFFICE.ROOT}`}>
        Back to Home Page
      </Button>
    </div>
  )
}
