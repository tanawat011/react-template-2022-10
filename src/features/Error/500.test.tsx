import { render } from '@testing-library/react'

import { Error500 } from './500'

describe('<Error500 />', () => {
  test('match snapshot', () => {
    const { asFragment } = render(<Error500 />)
    expect(asFragment()).toMatchSnapshot()
  })
})
