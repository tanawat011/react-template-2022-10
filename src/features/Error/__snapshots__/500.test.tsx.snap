// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Error500 /> match snapshot 1`] = `
<DocumentFragment>
  <div>
    500 Internal Server Error
  </div>
</DocumentFragment>
`;
